design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+    payment_info: NegotiableQuotePaymentInfo! @doc(description: "Negotiable quote payment info.")
+    available_shipping_methods: [ShippingMethodsOutput]! @doc(description: "Available shipping methods")
+}
+
+input NegotiableQuoteFilterInput {
+    ids: FilterEqualTypeInput @doc(description: "Filter Customer Negotiable quotes with an negotiable quote ID or list of negotiable quote IDs")
+    name: FilterMatchTypeInput @doc(description: "Filter by display name of the negotiable quote")
+}
+
+type NegotiableQuoteItem {
+    id: ID!
+    quantity: Float!
+    product: ProductInterface!
+}
+
+type AttachmentContent @doc(description: "Negotiable quote attachment file") {
+    base64_encoded_data: String!
+    mime_type: String!
+    name: String!
+}
+
+type NegotiableQuoteComment {
+    entity_id: ID!
+    parent_id: ID!
+    creator_type: Int
+    is_decline: Int
+    is_draft: Int
+    creator_id: ID!
+    comment: String
+    created_at: String
+    attachments: [NegotiableQuoteCommentAttachments]
+}
+
+type NegotiableQuoteCommentAttachments {
+    attachment_id: ID!
+    comment_id: ID
+    file_name: String
+    file_path: String
+    file_type: String
+}
+
+type NegotiableQuotePaymentInfo {
+    payment_methods: [PaymentMethodsOutput]! @doc(description: "All list of payment options and totals")
+    totals: TotalsOutput! @doc(description: "List of totals")
+}
+
+type PaymentMethodsOutput {
+    code: String @doc(description: "Payment method code")
+    title: String @doc(description: "Payment method title")
+}
+
+type TotalsOutput {
+    grand_total: Money
+    subtotal_including_tax: Money
+    subtotal_excluding_tax: Money
+    discount: CartDiscount
+    subtotal_with_discount_excluding_tax: Money
+    applied_taxes: [CartTaxItem]
+}
+
+type CartTaxItem {
+    amount: Money!
+    label: String!
+}
+
+type CartDiscount {
+    amount: Money!
+    label: [String!]!
+}
+
+type RequestNegotiableQuoteOutput {
+    negotiable_quote:NegotiableQuote!
+}
+
+input FileInput @doc(description: "The list of file attachment codes") {
+    base64_encoded_data: String!
+    mime_type: String!
+    name: String!
+}
+
+input NegotiableQuoteShippingAddressInput {
+    firstname: String! @doc(description: "The first name of the person associated with the billing address")
+    lastname: String! @doc(description: "The family name of the person associated with the billing address")
+    telephone: String! @doc(description: "The telephone number")
+    street: [String]! @doc(description: "A list of strings that define the street number and name")
+    city: String! @doc(description: "The city or town")
+    region: String! @doc(description: "The region name")
+    region_id: Int @doc(description: "The region ID")
+    postcode: String! @doc(description: "The customer's ZIP or postal code")
+    country_code: String! @doc(description: "The country code")
+}
+
+type ShippingMethodsOutput {
```

Comment:
There is an existing type `AvailableShippingMethod`, what is the purpose create a new one?
```suggestion
```